pkg/controllers/netpol/network_policy_controller.go

```diff
@@ -377,6 +381,38 @@ func (npc *NetworkPolicyController) ensureTopLevelChains() {
 		ensureRuleAtPosition(kubeInputChainName, whitelistServiceVips, uuid, externalIPIndex+4)
 	}
 
+	// for the traffic to/from the local pod's let network policy controller be
+	// authoritative entity to ACCEPT the traffic if it complies to network policies
+	for _, chain := range chains {
+		comment := "rule to explictly ACCEPT traffic that comply to network policies"
+		args := []string{"-m", "comment", "--comment", comment, "-m", "mark", "--mark", "0x20000/0x20000", "-j", "ACCEPT"}
+		err = iptablesCmdHandler.AppendUnique("filter", chain, args...)
```

Comment:
I take it that the fact that we only accept our own traffic (i.e. traffic to/from pods) and don't comment on other traffic is purposeful right? i.e. the user can make their own firewall policy decisions for traffic that is not k8s based?